File: 04 Technical/02 Server/01 Logics/01 Python/01 General/06 Factorial 2.py
# Factorial

# Function
def findFactorial(num):
    if num <= 0:
        return 1
    output = num * findFactorial(num-1)
    return output

given_number = 5  # int(input())
print(given_number)

print("----------------")

factorial = findFactorial(given_number)
print(factorial)

print("==================")

File: 04 Technical/01 Client/01 Logics/01 Javascript/06 Factorial 2.js
// Factorial

//  Functions
function findFactorial(num) {
	if (num <= 1) {
		// Base
		return num;
	}
	result = num * findFactorial(num - 1);
	return result;
}

let given_number = 5;
console.log(given_number);

console.log('---------------');

let factorial = findFactorial(given_number);
console.log(factorial);

console.log('=====================');
